vm/scripts/customize-vm

```diff
@@ -20,7 +20,7 @@
 systemctl disable NetworkManager-wait-online
 systemctl disable sshd
 
-grubby --update-kernel=ALL --args="default_hugepagesz=1GB hugepagesz=1G hugepages=8 isolcpus=2-7"
+grubby --update-kernel=ALL --args="default_hugepagesz=1GB hugepagesz=1G hugepages=1 isolcpus=2-7"
```

Comment:
Shouldn't `hugepages=1` be `hugepages=4`?

Reply:
It's still under inquiry, but in the the guest we only get 1 hugepages allocated by  HugeTLB.